how-to.html

```diff
@@ -0,0 +1,80 @@
+    <ul>
+      <li>
+        <strong>Know your hardware options</strong>. If you have a smart phone, there are a lot of voice memo and similar
+        apps out there to turn your phone into a recording booth. If you have a webcam, your webcam likely has audio
+        recording capability. If you have a laptop with a built in camera, you likely can use your laptop to record as well
+        though you are more likely to pick up the sounds of your laptop's fans, etc.
+      </li>
+      <li>
+        <strong>Record in a high-fidelity format</strong>. We accept MP3s, but saving your recording as a WAV or AIFF <fieldset>
+          will help a lot in preserving the quality of your recording.
+      </li>
+      <li>
+        <strong>Record in a quiet space</strong>. Even things like your fan, refrigerator, pet koala, can get picked
+        up easily when recording.
+      </li>
+      <li>
+        <strong>Calibrate your recording space using headphones</strong>. Listening to your microphone through your headphones lets you determine if
+        you need to move to another environment, and whether or not your volume levels are appropriate.
+      </li>
+      <li>
+        <strong>Don't directly position your microphone</strong>. If you do, your "P" and "B" sounds will likely be
+        too loud. Setting your microphone at a small angle from in front of you can help cut this down.
+      </li>
+      <li>
+        <strong>Muffle what's behind you</strong>. The microphone will pick up all sound that is directly behind you. While
+        sitting a few feet from a wall or corner is best to cut down on background sound, muffling that area is even better.
+        Draping a bedspread or quilt behind you can cut down a lot on small echoes in the room, focusing your voice more.
+      </li>
+      <li>
+        <strong>Speak consistently</strong>. If your recording is consistent in volume, it will be easier to edit. Tell your
+        story with an assertive volume; otherwise, when mixed, whispering will cause background sounds to be amplified.
+      </li>
+    </ul>
+  </p>
+  <h2>Notes on Usage</h2>
+  <p>
+    All recording submitted remain property of their authors, but are released to our usage
+    under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.
+    This retains you as the owner of the original content, requires all usage include attribution to you,
+    and allows us to master your recordings and to distribute your recording as part of our podcast.
+  </p>
+  <p>
+    By submitting your recording, you agree to allow your recording to be used in an upcoming episode, including compilations
+    or other recordings outside our scheduled episodes. You will be attributed fully during the podcast and in related materials,
+    and if you need to protect your own privacy we can attribute your recording either anonymously or under a pen name.
+  </p>
+</div>
```

Comment:
You know you can write Jekyll pages in Markdown, right?

Reply:
Admittedly we wouldn't have the `rel` attributes then, but we would have smart punctuation :p

Reply:
I am aware, but I prefer the control of html, plus the `rel` is important to avoid spoofing =)